WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderActivityLauncher.java

```diff
@@ -255,9 +283,15 @@ private static void openUrlInternal(Context context, @NonNull String url) {
      */
     private static void openUrlExternal(Context context, @NonNull String url) {
         try {
+            // disable deeplinking activity so to not catch WP URLs
+            ActivityLauncher.disableDeepLinking(context);
```

Comment:
It might make sense to use `try...finally` here, putting `enableDeepLinking` in the `finally` block. Otherwise deep linking won't be re-enabled if something goes wrong.